walletrpc/service.proto

```diff
@@ -8,18 +8,32 @@ option go_package = "lightwalletd/walletrpc";
 option swift_prefix = "";
 import "compact_formats.proto";
 
+// An identifier for a Zcash value pool.
+enum PoolType {
+    POOL_TYPE_UNKNOWN = 0;
```

Comment:
What should `GetBlockRange` return if this pool type is specified? I think maybe this value should be eliminated (have only the other three values).

Reply:
The problem is that for protobuf enums, a value can be default-filled, so we need the default value to be something that can't be confused with a valid pool type. If this type is ever encountered, the server should return an error.